src/android-verify.js

```diff
@@ -6,23 +6,40 @@ var request = require('isomorphic-fetch');
 var asn = require('asn1.js');
 const jsonSettings = require('../settings/settings.json');
 const R = require('ramda');
-import {AndroidProof} from '../certs/AndroidProof.js';
 // $FlowFixMe
 const Buffer = require('buffer').Buffer;
 
 
-export const verify = async (data: Uint8Array) => {
+export const verify = async (data: Uint8Array, version: string) => {
```

Comment:
Here is better to use a declared type over a generic one `type AndroidVersion = 'v1' | 'v2'`. Not a blocker

Reply:
Good to know! I'll keep that in mind for future developments, thanks!